### PR TITLE
Updated the 2017 conference details

### PR DESCRIPTION
**Pull request by swill**
Added Miami, removed the broken link for Brazil and referenced the correct video playlist for the Montreal event.

### CODE REVIEW

**Comment by swill** [comment]
@pdion891 and @rhtyd you guys mind reviewing / approving these changes?

**Comment by rafaelweingartner** [comment]
LGTM

**Comment by pdion891** [comment]
Can you take the oportunity to upgrade the Year in this page too ?
https://github.com/apache/cloudstack-www/blob/master/source/layouts/layout.erb#L131

**Comment by pdion891** [comment]
otherwise, LGTM

**Comment by swill** [comment]
You mean the Miami event? That is the one that is being done at apachecon.
I intentionally left it out so they would go to the CCC website to get
details since there are additional details there we want people to see.

Does that make sense?

**Comment by swill** [comment]
@pdion891 I updated the copyright date as you requested.

File: source/index.html.erb [hunk 1]
```diff
@@ -113,22 +113,23 @@ Join Us
 
 		<div class="panel-body" id="events">
 
-			<h4>2016 CloudStack Collaboration Conferences</h4>
+			<h4>2017 CloudStack Collaboration Conferences</h4>
 
 			<p>
-			In 2016, two Apache <a href="http://cloudstackcollab.org/">CloudStack Collaboration Conferences</a> events will be held in the Americas. Each event will feature morning plenary sessions, afternoon breakout sessions in user and developer tracks and a hackathon. Learn more at: 
+			In 2017, two Apache <a href="http://cloudstackcollab.org/">CloudStack Collaboration Conference</a> events will be held in the Americas. Each event will feature a lot of opportunities to learn, collaborate and engage with the CloudStack community.  The CloudStack Collaboration Conferences are valuable for both developers and users, and are a great way to get involved with the community.
```
Comment (terbolous) [inline source/index.html.erb]: No, not really. This section says 'In 2017, two Apache CCC events ....' and points to a web page where only one of the two events is mentioned, but also mentions a previous (Canada) event.

Personally it is likely that I would click on the first link and expect to find the ApacheCon there, even if only a link to the proper ApacheCon site.

It is really more about what content exists on cloudstackcollab.org, so I'm fine with merging this one and discussing cloudstackcollab elsewhere.

Comment (swill) [inline source/index.html.erb]: sorry, I just fixed that.  jenkins pushed an old version of the site.  thanks for pointing that out.

Comment (swill) [inline source/index.html.erb]: I removed the Canada event and made sure that only Miami and Brazil showed on that site now.  Thanks...

Comment (terbolous) [inline source/index.html.erb]: Looks good, I have nothing more :-) LGTM

**Comment by pdion891** [comment]
+1

**Comment by swill** [comment]
I fixed some malformed html that was causing the security page to render incorrectly.  I did not change any content in that commit, so I am going to assume that a fix of that nature does not need to be approved.  I am going to push this live now so we have the website correctly representing the Miami CCC event.

**Pull request merged**